File: Unidad 5 C#/Programa2U5.cs
using System;

namespace Programa2U5
{
    internal class Programa2U5
    {
        static void Main(string[] args)
        {
            int opcion;
            Console.WriteLine("Elige el tipo de conversion a realizar: ");
            Console.WriteLine("1. Temperatura °C a °F");
            Console.WriteLine("2. Temperatura °F a °K");
            Console.WriteLine("3. Medida pulgadas a metros");
            Console.WriteLine("4. Tiempo horas a segundos");
            opcion = Convert.ToInt32(Console.ReadLine());

            switch(opcion)
            {
                case 1:
                    int c;
                    Console.Write("Escribe los °C a convertir: ");
                    c= Convert.ToInt32(Console.ReadLine());
                    Console.WriteLine("Tu resultado es: " + temperaturaCF(c));
                    break;
                case 2:
                    Console.WriteLine("Tu resultado es: "+temperaturaFK());
                    break;
                case 3:
                    Medida();
                    break;
                case 4:
                    Console.WriteLine(tiempo());
                    break;
                default: 
                    Console.WriteLine("La opción es incorrecta");
                    break;
            }//FIN SWITCH
        }//FIN MAIN
        public static double temperaturaCF(double c)
        {
            double resultado = (c * 1.8) + 32;
            return resultado;
        }
        public static double temperaturaFK()
        {
            double F, resultado;
            Console.Write("Escribe la temperatura °F a convertir: ");
            F= Convert.ToDouble(Console.ReadLine());
            resultado = (F - 32) * (5 / 9) + 273.15;
            return resultado;
        }
        public static void Medida()
        {
            double p, resultado;
            Console.Write("Escribe la medida en pulgadas: ");
            p= Convert.ToDouble(Console.ReadLine());
            resultado = p / 39.37;
            Console.WriteLine("Tu resultado es: " + resultado);

        }
        public static double tiempo()
        {
            double h;
            Console.Write("Escribe las horas a convertir: ");
            h = Convert.ToDouble(Console.ReadLine());
            return h * 3600;
        }
    }
}

File: Unidad 5 Java/Programa2U5.java
package com.mycompany.programa2U5;
import java.util.Scanner;

public class Programa2U5 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int opcion;

        System.out.println("Elige el tipo de conversion a realizar: ");
        System.out.println("1. Temperatura °C a °F");
        System.out.println("2. Temperatura °F a °K");
        System.out.println("3. Medida pulgadas a metros");
        System.out.println("4. Tiempo horas a segundos");

        opcion = scanner.nextInt();

        switch (opcion) {
            case 1:
                double c;
                System.out.print("Escribe los °C a convertir: ");
                c = scanner.nextDouble();
                System.out.println("Tu resultado es: " + temperaturaCF(c));
                break;
            case 2:
                System.out.println("Tu resultado es: " + temperaturaFK());
                break;
            case 3:
                medida();
                break;
            case 4:
                System.out.println(tiempo());
                break;
            default:
                System.out.println("La opción es incorrecta");
                break;
        }
    }

    public static double temperaturaCF(double c) {
        return (c * 1.8) + 32;
    }

    public static double temperaturaFK() {
        double F, resultado;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Escribe la temperatura °F a convertir: ");
        F = scanner.nextDouble();
        resultado = (F - 32) * (5.0 / 9.0) + 273.15;
        return resultado;
    }

    public static void medida() {
        double p, resultado;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Escribe la medida en pulgadas: ");
        p = scanner.nextDouble();
        resultado = p / 39.37;
        System.out.println("Tu resultado es: " + resultado);
    }

    public static double tiempo() {
        double h;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Escribe las horas a convertir: ");
        h = scanner.nextDouble();
        return h * 3600;
    }
}